libsel4vm/src/arch/arm/vgic/vgic.c

```diff
@@ -182,6 +182,7 @@ struct gic_dist_map {
     uint32_t component_id[4];                           /* [0xFF0, 0xFFF] */
 };
 
+#define NUM_LRS 4
```

Comment:
To anser https://github.com/seL4/seL4_projects_libs/pull/28#issuecomment-1063571957, yes, would be nice to have a comment here where the value comes from.

Reply:
Done.